### DISCUSSION TITLE
Astroid frontend css file for the content editor

### DISCUSSION THREAD

**Discussion by franke12359** [templaza/astroid-framework#811] [Ideas]
It is possible to get a actual frontend css file as editor.css file from Astroid system if the user press the "clear-cache" button?
Or is there anywhere a css file with the actual Astroid frontend settings for eg. the jce editor?

**Comment by Chacapamac** [comment]
**JCE - ASTROID WYSIWYG**

◉ Add a custom css file only use by the JCE edition Window __________________________________
media ➔ templates ➔ site ➔ Your_Astroid_Template ➔ css ➔ editor_content.css

• Set the Editor Global Configuration “Styles” option to “Template CSS File”, so the editor can load the correct compiled css files for the Astroid framework
![Screen Shot 2024-09-21 at 11 07 12 AM](https://github.com/user-attachments/assets/e292fb0f-c5ab-49a8-b29d-f23cf94e61b1)


• Go to JCE “Edit Profile”  ➔ “Editor Parameters” tab and
— Change “Editor Styles” to “Add”
![Screen Shot 2024-09-21 at 11 07 54 AM](https://github.com/user-attachments/assets/98fb061d-6599-482f-b54c-acd195806496)

— and add the path to your newly created custom editor css file  editor_content.css
media/templates/site/$template/css/editor_content.css
![Screen Shot 2024-09-21 at 11 09 12 AM](https://github.com/user-attachments/assets/dea24243-a553-462f-8c56-ef948d2fc6f1)


In this file you will be able to load local Astroid Font Awesome css to preview icons, fine tune how your editor window will look like and even add elements that are visible ONLY in the editor window.


**My discussion with JCE**
https://www.joomlacontenteditor.net/support/forum/editor/107409-last-version-of-jce-%E2%80%94-t4-compiled-css-use-automatically-to-style-editor-can-i-use-in-astroid-framework-only-one-main-compiled-css-file#reply-107452
